source/constructs/api/data_source/crud.py

```diff
@@ -260,6 +315,22 @@ def delete_rds_connection(account: str, region: str, instance: str):
     session.merge(rds_instance_source)
     session.commit()
 
+def delete_jdbc_connection(provider: str, account: str, region: str, instance: str):
+    session = get_session()
+    jdbc_instance_source = session.query(JDBCInstanceSource).filter(JDBCInstanceSource.instance_id == instance,
+                                                                  JDBCInstanceSource.region == region,
+                                                                  JDBCInstanceSource.aws_account == account,
+                                                                  JDBCInstanceSource.account_provider == provider).order_by(
+        desc(RdsInstanceSource.detection_history_id)).first()
+    jdbc_instance_source.glue_database = None
+    jdbc_instance_source.glue_crawler = None
+    jdbc_instance_source.glue_connection = None
+    jdbc_instance_source.glue_vpc_endpoint = None
+    jdbc_instance_source.glue_crawler_last_updated = datetime.datetime.utcnow()
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Naive datetime objects are treated by many datetime methods as local times, We recommend that you create aware timestamps with the `now` or `fromtimestamp` functions using the `tzinfo` to attribute to represent times in UTC. Do not use the [datetime.utcnow()](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow) because it returns only a `naive` datetime object. 

[Learn more](https://docs.python.org/3/library/datetime.html#aware-and-naive-objects)

Reply:
Will be resolved in later version.